### PR TITLE
nested passenger info in bootstrap card

### PR DESCRIPTION
**Pull request by CFly17**
This is what it looks like now (button not yet functional): 
image

image

### CODE REVIEW

**Review by inregards2pluto — APPROVED** [review]
That looks great!! Nice job :)

**Review by KiraLivingston — COMMENTED** [review]
Looking good!

File: webapp/app.py [hunk 1]
```diff
@@ -178,7 +170,36 @@ def getpassengers():
...
+    if random_passenger[4]=='C':
+        random_passenger[4]='Cherbourg, France'
+    if random_passenger[4]=='S':
+        random_passenger[4]='Southampton, England'
+    if random_passenger[4]=='Q':
+        random_passenger[4]='Queenstown, Ireland'
+    if random_passenger[4]=='B':
+        random_passenger[4]='Belfast, Ireland'
+    
+    if random_passenger[6] is None:
```
Comment (pizzahut111) [inline webapp/app.py:187]: if ticketNumber was null the browser didn't know how to format it.

File: webapp/app.py [hunk 2]
```diff
@@ -178,7 +170,36 @@ def getpassengers():
...
+        random_passenger[6] ='N/A'
+    else: 
+        random_passenger[6]="{:,.0f}". format(random_passenger[6])
+    
+    if random_passenger[7] is None:
+        random_passenger[7] ='N/A'
```
Comment (CFly17) [inline webapp/app.py:193]: Also fixed browser so it can render null values for ticket price

File: webapp/app.py [hunk 3]
```diff
@@ -178,7 +170,36 @@ def getpassengers():
...
+        random_passenger[4]='Cherbourg, France'
+    if random_passenger[4]=='S':
+        random_passenger[4]='Southampton, England'
+    if random_passenger[4]=='Q':
+        random_passenger[4]='Queenstown, Ireland'
+    if random_passenger[4]=='B':
+        random_passenger[4]='Belfast, Ireland'
+    
+    if random_passenger[6] is None:
+        random_passenger[6] ='N/A'
```
Comment (CFly17) [inline webapp/app.py:188]: if ticket number was null the browser didn't know how to format it. Here is the simple 'if' logic to fix it

**Review by lewars1108 — APPROVED** [review]

**Review by PGrickswim — APPROVED** [review]

**Pull request merged**